xarray_filters/tests/test_reshape.py

```diff
@@ -153,6 +157,51 @@ def test_from_features_dropped_rows(X):
     data2 = features.from_features()
 
     # Assert that the coords are correct, and NaNs are in the right places
-    if np.nan in data2.to_array()[0]:
+    if np.nan in data2.to_xy_arrays()[0]:
         assert np.array_equal(data2.coords.to_index().values, data1.coords.to_index().values)
-        assert np.allclose(data2.to_array()[0], zerod_vals_copy, equal_nan=True)
+        assert np.allclose(data2.to_xy_arrays()[0], zerod_vals_copy, equal_nan=True)
+
+dsets = [xr.Dataset(x) for x in Xs]
+np_arrs = [x.to_features().features.values for x in Xs]
+dfs = [pd.DataFrame(x.to_features().features.values) for x in Xs]
+choices = {
```

Comment:
Note here we eventually (2018) want to test eventually any acceptable `X` or `y` data structures when `to_xy_arrays` is used as a standalone function, e.g. pandas, numpy, dask, DataArray/Dataset/MLDataset.  The test is a parameterized one set up for that, but currently is only testing MLDataset X and numpy y.